admin/class-boldgrid-backup-admin-jobs.php

```diff
@@ -162,10 +182,11 @@ public function post_archive_files( $info ) {
 		}
 
 		$args = array(
-			'filepath'    => $info['filepath'],
-			'action'      => 'boldgrid_backup_post_jobs_email',
-			'action_data' => $info,
-			'post_action' => 'delete_all_prior',
+			'filepath'     => $info['filepath'],
+			'action'       => 'boldgrid_backup_post_jobs_email',
+			'action_data'  => $info,
+			'post_action'  => 'delete_all_prior',
+			'action_title' => __( 'Send an email after all jobs have been ran', 'boldgrid-backup' ),
```

Comment:
The pro plugin is adding jobs too. Instead of having to update all those too, maybe just use the `post_action` as the title if the `action_title` is not set.

Reply:
I will double check, but I searched for all calls to the method for adding a job, and the only one missing a title was this one.